okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/IoExceptionResponseHandler.java

```diff
@@ -38,8 +40,9 @@
             String body = response.body() != null && response.body().byteStream() != null
                     ? toString(response.body().byteStream())
                     : "<empty>";
-            return Optional.of(new IOException(String.format("Error %s. "
-                    + "(Failed to parse response body as SerializableError.) Body:%n%s", response.code(), body)));
+            return Optional.of(new SafeIoException(
+                    String.format("Error %s. (Failed to parse response body as SerializableError.) ", response.code()),
```

Comment:
Lets not use string format anymore, instead just having "Failed to parse response body as SerializableError".

Reply:
@justinuang I think your suggestion makes sense once we've changed our internal logging implementation to pull out the args, but until that's implemented I think it's OK to run with the String.format!